client/src/pages/Settings/Availability/FormInterface.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react';
+import { Typography } from '@mui/material';
+import Grid from '@mui/material/Grid';
+import FormGroup from '@mui/material/FormGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Checkbox from '@mui/material/Checkbox';
+import Select from '@mui/material/Select';
+import MenuItem from '@mui/material/MenuItem';
+
+interface GenerateFormInterface {
+  day: string;
+}
+
+const GenerateFormInterface: React.FC<GenerateFormInterface> = ({ day }) => {
+  const [from, setFrom] = useState('10');
+  const [to, setTo] = useState('22');
+  const [checked, setChecked] = useState(true);
+
```

Comment:
Could we utilize Formik instead? Take a look at the Sign up page or even the EditProfile page for an example of how to use Formik.

Reply:
Added Formik to manage states. Hope the new version makes sense! I aimed to keep the UI the same as before.